libafl_targets/src/sancov_8bit.rs

```diff
@@ -208,17 +221,9 @@ mod observers {
             res
         }
 
-        fn hash(&self) -> u64 {
-            let mut hasher = RandomState::with_seeds(0, 0, 0, 0).build_hasher();
-            for map in unsafe { &*addr_of!(COUNTERS_MAPS) } {
-                let slice = map.as_slice();
-                let ptr = slice.as_ptr();
-                let map_size = slice.len() / core::mem::size_of::<u8>();
-                unsafe {
-                    hasher.write(from_raw_parts(ptr, map_size));
-                }
-            }
-            hasher.finish()
+        #[inline]
+        fn hash_simple(&self) -> u64 {
+            RandomState::with_seeds(0, 0, 0, 0).hash_one(self)
```

Comment:
We _could_ use the hash_std method here that can use xxh3 (but probably we don't need to) https://github.com/AFLplusplus/LibAFL/blob/8bce605503af86692e00f5d0ca9d2e72b6efa919/libafl_bolts/src/lib.rs#L283

Reply:
Why does this method even exist? We should just have a `type` definition for StdHash or something, probably.

Reply:
Not even sure if it implements any rust traits tbh

Reply:
`hash_simple` does not implement any rust traits. If hash_std is okay then can use that instead. I don't really like it but `Hash` trait in rust doesn't seem to provide any default hashing function, so I made this

Reply:
Generally, we don't actually _want_ a default hashing function. This should be specified by the user, which is why we wanted the `Hash` impl (as this allows us to use a generic `Hasher`).